packages/debian/distributions

```diff
@@ -0,0 +1,7 @@
+Origin: Scality
+Label: metalk8s
+Suite: stable
+Codename: bionic
+Architectures: amd64
+Components: main metalk8s-bionic metalk8s-bionic-backports metalk8s-bionic-updates metalk8s-kubernetes-xenial metalk8s-salt_ubuntu1804 metalk8s-scality
```

Comment:
Why not `metalk8s-main`?

Reply:
In fact I have to delete main... there is no need of this component